### DISCUSSION TITLE
Unable to create new database on version 3.4.4

### DISCUSSION THREAD

**Discussion by GiovanniRepository** [pawelsalawa/sqlitestudio#5004] [General]
### Details
When i click on Add database form database menu or on dedicated icon it allow my only to load existing database, not to create new database. 

### Operating system
Windows 11

### SQLiteStudio version
My version of SQLiteStudio is 3.4.4 and I verified that in version 3.3.3 it was possible to create a database in the exact same way. I attach the photos here, one of version 3.3.3, and the other of my version, 3.4.4
Let me know when the problem has been resolved. Thank you

![Screenshot version 3 3 3](https://github.com/pawelsalawa/sqlitestudio/assets/169991381/4b6e7e68-5c6e-4874-8c48-72de03f441a3)
![Screenshot version 3 4 4](https://github.com/pawelsalawa/sqlitestudio/assets/169991381/8df155ff-9431-4c77-8f68-32152c70ac0b)

**Comment by pawelsalawa** [comment]
Version 3.4 uses just one icon/button for both operations. Just use the one you see to create new database.

**Reply by GiovanniRepository** [reply]
Clicking on the yellow button shown in the photo only allows to import existing databases, not to create a new database.

**Comment by andreealucaci63** [comment]
Hi GiovanniRepository. Have you managed to find a solution? I have the same trouble.

**Reply by GiovanniRepository** [reply]
Hi Andreealucaci63, the only solution that was recommended to me is to use a Mac keyboard key combination to remove the full screen on the Mac when app UTM crashes. Try with CMD+Ctrl+F. Another solution is to avoid making the UTM app full screen, and when it crashes force close the application. I hope these tips are useful to you

**Comment by andreealucaci63** [comment]
Thank you
Kind regards
Andreea Lucaci
